Jenkinsfile

```diff
@@ -1,9 +1,10 @@
 #!groovy
 
-def push_image(tag) {
+// tag image, push to repo, remove local tagged image
+def tag_image_as(tag) {
   script {
-    def image = docker.image("${env.DOCKER_REGISTRY}/${env.DOCKER_IMAGE}:${env.COMMIT_HASH}")
-    image.push(tag)
+    docker.image("${DOCKER_IMAGE_URL}:${env.COMMIT_HASH}").push(tag)
```

Comment:
Is `DOCKER_IMAGE_URL` available here, or should this be `env.DOCKER_IMAGE_URL`?

Reply:
Yes, it is available there without the `env.` prefix.